src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Security/TokenIntrospection.feature.disabled

```diff
@@ -0,0 +1,152 @@
+
+        Scenario: Reject a Resource endpoint DELETE request where the token is from the wrong issuer
+            Given a JSON web token with
+                  | issuer              | expiresAt  |
+                  | https://example.com | 2014699317 |
+                  # 2014699317 is in the year 2033
+             When sending a DELETE request with a token to "/ed-fi/academicWeeks/replaceWithAMoreRealisticId"
+             Then it should respond with 401
+
+        Scenario: Reject a Resource endpoint GET request where the token is expired
+            Given a JSON web token with
+                  | issuer           | expiresAt  |
+                  | http://localhost | 1683076917 |
+                  # 1683076917 is in the year 2023
+             When sending a GET request with a token to "/ed-fi/academicWeeks"
+             Then it should respond with 401
+
+        Scenario: Reject a Resource endpoint POST request where the token is expired
+            Given a JSON web token with
+                  | issuer           | expiresAt  |
+                  | http://localhost | 1683076917 |
+                  # 1683076917 is in the year 2023
+             When sending a POST request with a token to "/ed-fi/academicWeeks"
+             Then it should respond with 401
+
+        Scenario: Reject a Resource endpoint PUT request where the token is expired
+            Given a JSON web token with
+                  | issuer           | expiresAt  |
+                  | http://localhost | 1683076917 |
+                  # 1683076917 is in the year 2023
+             When sending a PUT request with a token to "/ed-fi/academicWeeks/replaceWithAMoreRealisticId"
+             Then it should respond with 401
+
+        Scenario: Reject a Resource endpoint DELETE request where the token is expired
+            Given a JSON web token with
+                  | issuer           | expiresAt  |
+                  | http://localhost | 2014699317 |
+                  # 2014699317 is in the year 2033
+             When sending a DELETE request with a token to "/ed-fi/academicWeeks/replaceWithAMoreRealisticId"
+             Then it should respond with 401
+
+        Scenario: Accept a Resource endpoint GET request where the token is valid
+            Given a JSON web token with
+                  | issuer           | expiresAt  |
+                  | http://localhost | 2014699317 |
+                  # 2014699317 is in the year 2033
+             When sending a GET request with a token to "/ed-fi/academicWeeks"
+             Then it should respond with 200
+
+        Scenario: Accept a Resource endpoint POST request where the token is valid
+            Given a JSON web token with
+                  | issuer           | expiresAt  |
+                  | http://localhost | 2014699317 |
+                  # 2014699317 is in the year 2033
+             When sending a POST request with a token to "/ed-fi/academicWeeks"
+             Then it should respond with 400
```

Comment:
Is this a 400 because the post does not contain body?

Reply:
Yes. I will clarify with a comment